provider/infoblox/infoblox.go

```diff
@@ -327,7 +363,34 @@ func (p *InfobloxProvider) findZone(zones []ibclient.ZoneAuth, name string) *ibc
 	return result
 }
 
+func (p *InfobloxProvider) findReverseZone(zones []ibclient.ZoneAuth, name string) *ibclient.ZoneAuth {
```

Comment:
@cgroschupp can you please add test coverage for `findReverseZone` function, similar to `TestInfobloxZones`